pkg/reconciler/topology/partitionset/partitionset_reconcile.go

```diff
@@ -97,7 +99,11 @@ func (c *controller) reconcile(ctx context.Context, partitionSet *topologyv1alph
 	} else {
 		matchLabelsMap = partition(shards, dimensions, nil)
 	}
-	partitionSet.Status.Count = uint16(len(matchLabelsMap))
```

Comment:
can we just make this an uint?

Reply:
should not be a problem...but then why not int? what is the advantage of the uint?

Reply:
Negative numbers of shards doesn't make any sense either :)

Reply:
yeah, sure...but then another convert is needed from uint to int. In this case `len()` should not return a negative number ;)

Reply:
ok, changed to uint